configuration/config-txt/memory.md

```diff
@@ -2,7 +2,7 @@
 
 ## gpu_mem
 
-Specifies how much memory, in megabytes, to reserve for the exclusive use of the GPU: the remaining memory is allocated to the ARM CPU. The default value is `76`, except on the Pi Zero and Pi 1 where it is `64`. You should set `gpu_mem` to the lowest possible value to ensure best performance. If a particular graphics feature is not working correctly, try increasing the value of `gpu_mem`, being mindful of the recommended maximums shown below.
+Specifies how much memory, in megabytes, to reserve for the exclusive use of the GPU: the remaining memory is allocated to the ARM CPU. For Pis with less than 1GB of memory, the default is `64`; for Pis with 1GB or more of memory the default is `76`. To ensure best performance, you should set `gpu_mem` to the lowest possible value. If a particular graphics feature is not working correctly, try increasing the value of `gpu_mem`, being mindful of the recommended maximums shown below.
```

Comment:
I wonder if it's worth rewording "To ensure best performance, you should set `gpu_mem` to the lowest possible value." to something like "To ensure best performance, you should allocate as much RAM as possible to the ARM CPU, by setting `gpu_mem` to the lowest possible value." ? :shrug:

Reply:
More words for the same meaning.

Reply:
Meh, I just thought it might be nice to explain _why_ selecting a low `gpu_mem` value "ensures best performance". But I'm not bothered if you think it's too superfluous.

Reply:
I've added your suggestion, with some rejigging and adjustment to the wording. It is somewhat wordy, but we've had folk on the forums query things, and get the wrong idea in the past, so I think the level of detail we've got here is probably justified.